Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUCompositorIntegrationImpl.cpp

```diff
@@ -42,38 +42,38 @@ CompositorIntegrationImpl::CompositorIntegrationImpl(ConvertToBackingContext& co
 
 CompositorIntegrationImpl::~CompositorIntegrationImpl() = default;
 
-#if PLATFORM(COCOA)
-Vector<MachSendRight> CompositorIntegrationImpl::getRenderBuffers()
+void CompositorIntegrationImpl::prepareForDisplay()
 {
-    return m_renderBuffers.map([] (const auto& renderBuffer) {
-        return MachSendRight::adopt(IOSurfaceCreateMachPort(renderBuffer.get()));
-    });
+    // FIXME: https://bugs.webkit.org/show_bug.cgi?id=250993 Wait for the results to be fully drawn
```

Comment:
can we have this member function take a completion handler as it will eventually need to be asynchronous?